### ISSUE TITLE
Streamline the release version in scripts and filenames

### ISSUE THREAD

**Issue by dleehr** [matthewhirschey/ddh.org#6]
The R scripts all currently set the release version as `19Q3`, and the makefile proposed in #5 hard-codes it in the expected filenames. While it may seem simple to change this in the code, the best practice would be to provide it as a variable when generating the data files, and not have it be hard-coded in scripts and Makefiles that are otherwise reusable.

We don't need an immediate solution, but a good use of issues is to suggest and discuss solutions, so let's :)

**Comment by matthewhirschey** [comment]
Agree.  I thought the same thing when making the R scripts (I'll need to change each when updating), but need to think about a way to make a variable populate several files. 

Something like loading an R variable as a parameter that can be defined by either an R script, or an R object...

**Comment by matthewhirschey** [comment]
One solution would be to take the release *out* of the parameter YAML header and call as a normal R object. In this case, each quarterly update would get a new `'current_release.Rds`' file, which would contain the `release` and the `url` for the data file. If `current_release.Rds` is updated, then the make file would re-run everything?

In this way, only a single file needs to be updated. Something like this @dleehr ?

e.g.: https://stackoverflow.com/questions/5577221/how-can-i-load-an-object-into-a-variable-name-that-i-specify-from-an-r-data-file

**Comment by matthewhirschey** [comment]
I generated a new file called current_release.R, which creates 5 new objects, previously in the YAML header. This creates objects which the other R scripts call. I added a line of code in the front of the generate_*.R scripts to call this file: source(current_release.R), and tested it locally. 

My thinking is that I can update this script with new information, which would then propagate to the other generate scripts to update that information. 

I think this closes this issue, but want to hear your feedback @dleehr and @johnbradley
